server/sonar-server/src/test/java/org/sonar/server/platform/db/EmbeddedDatabaseTest.java

```diff
@@ -50,7 +50,7 @@
   @Rule
   public TemporaryFolder temporaryFolder = new TemporaryFolder();
   @Rule
-  public Timeout timeout = Timeout.seconds(10);
+  public Timeout timeout = Timeout.seconds(30);
```

Comment:
Travis is our reference of "slow" and 10s is ok there
the problem is most likely not on the UT's side